primitives/src/lib.rs

```diff
@@ -326,15 +326,15 @@ pub enum PayoutState {
 	Finalized = 7,
 }
 
-pub const KEY_TYPE: KeyTypeId = KeyTypeId(*b"cer!");
+pub const DAC_VERIFICATION_KEY_TYPE: KeyTypeId = KeyTypeId(*b"cer!");
 
 pub mod sr25519 {
 	mod app_sr25519 {
 		use scale_info::prelude::string::String;
 		use sp_application_crypto::{app_crypto, sr25519};
 
-		use crate::KEY_TYPE;
-		app_crypto!(sr25519, KEY_TYPE);
+		use crate::DAC_VERIFICATION_KEY_TYPE;
+		app_crypto!(sr25519, DAC_VERIFICATION_KEY_TYPE);
```

Comment:
There is a similar pattern in `authority-discovery`, `babe`, and such. But they define `AuthorityId` as an alias for the `Public` made by the macros (https://github.com/paritytech/polkadot-sdk/blob/c8d2251cafadc108ba2f1f8a3208dc547ff38901/substrate/primitives/authority-discovery/src/lib.rs#L35). Should we do the same for validators accounts or the `AccountId` suits for us?